app/controllers/ProfileControllers.js

```diff
@@ -178,4 +178,129 @@ exports.getProfileAccount = function (req, res, next, ProfileAccountID) {
  */
 exports.showProfileAccount = function (req, res) {
     return res.jsonp(req.profileAccount);
-};
\ No newline at end of file
+};
+
+
+
+/**
+ * New version 2.0
+ */
+
+/**
+ * Find profile by id
+ * Note: This is called every time that the parameter :profileId is used in a URL. 
+ * Its purpose is to preload the profile on the req object then call the next function. 
+ */
+exports.getProfileById = function (req, res, next, id) {
+    //console.log('id => ' + ProfileID);
+    db.profile.find({
+        where: {
+            id: id
+        }
+    }).then(function (profile) {
+        if (!profile) {
+            return next(new Error('Failed to load ProfileId ' + id));
+        } else {
+            req.profile = profile;
+            return next();
+        }
+    }).catch(function (err) {
+        return next(err);
+    });
+};
+
+// Update Profile
+exports.updateProfile = function (req, res) {
+
+    // create a new variable to hold the article that was placed on the req object.
+    var profile = req.profile;
+
+    profile.updateAttributes({
```

Comment:
Please include address, image, password and etc update also.

Reply:
those fields will be added after that because is separated api